flutter_local_notifications_linux/lib/src/ffi/libc.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:ffi' as ffi;
+
+import 'dart:io' show Platform;
+
+/// FFI binding for C library
+class Libc {
+  /// Constructs an instance of [Libc].
+  factory Libc() => _libc;
+
+  Libc._() {
```

Comment:
I reduced the boilerplate, but I didn't combine this with `LinuxPlatformInfo`, as this would make FFI calls less testable . Currently POSIX is part of the tests and formally we check the FFI bindings correctness.